INSTALL.md

```diff
@@ -30,15 +30,12 @@ directory of the source code tree as ``$source_directory``.
 
 Make sure these are installed on your system before building OpenEXR:
 
-* OpenEXR requires CMake version 3.10 or newer (or autoconf on Linux systems).
-  - NB: CMake 3.12 is required for current PyIlmBase support
+* OpenEXR requires CMake version 3.10 or newer
+  - NB: CMake 3.12 is required for configuring to build static and
+    shared libraries in the same config (object library support)
```

Comment:
Should we just bump the minimum to 3.12? There are a bunch of other projects in the ecosystem that require at least 3.12, so it's not really like users are making due with 3.10 and we'd be upsetting the apple cart.